frontend/app/templates/components/skills-list-row-show.hbs

```diff
@@ -29,7 +29,7 @@
   {{if this.skill.portfolio this.skill.portfolio "-"}}
 </td>
 <td>
-  <a role="button" onclick={{this.startEditing}} class="edit-buttons skill-edit-button">
+  <a role="button" onclick={{this.startEditing}} class="edit-buttons skill-edit-button" style='{{unless this.isAdmin "-webkit-filter: grayscale(100%);"}}'>
```

Comment:
it probably makes more sense to add the css class conditionally than putting the style here. Furthermore I thinkg webkit filter is Firefox specific if i am not mistake, make sure to check that.